package.json

```diff
@@ -1,112 +1,96 @@
-  },
-  "resolutions": {
-    "**/@typescript-eslint/eslint-plugin": "^4",
-    "**/@typescript-eslint/parser": "^4",
-    "**/eslint": "^7",
-    "**/jest": "^26",
-    "**/prettier": "^2",
-    "**/ts-jest": "^26",
-    "**/typescript": "^4"
-  },
-  "jest": {
-    "preset": "ts-jest",
-    "testEnvironment": "node",
-    "transform": {
-      "node_modules/variables/.+\\.(j|t)sx?$": "ts-jest"
-    },
-    "transformIgnorePatterns": [
-      "node_modules/(?!variables/.*)"
-    ]
-  },
-  "peerDependencies": {
-    "react": ">=16"
+    "nx": "nx",
+    "start": "nx serve",
+    "build": "nx build",
+    "test": "nx test",
+    "lint": "nx workspace-lint && nx lint",
+    "e2e": "nx e2e",
+    "affected:apps": "nx affected:apps",
+    "affected:libs": "nx affected:libs",
+    "affected:build": "nx affected:build",
+    "affected:e2e": "nx affected:e2e",
+    "affected:test": "nx affected:test",
+    "affected:lint": "nx affected:lint",
+    "affected:dep-graph": "nx affected:dep-graph",
+    "affected": "nx affected",
+    "format": "nx format:write",
+    "format:write": "nx format:write",
+    "format:check": "nx format:check",
+    "update": "nx migrate latest",
+    "workspace-generator": "nx workspace-generator",
+    "dep-graph": "nx dep-graph",
+    "help": "nx help",
+    "prepare": "husky install"
   },
-  "lint-staged": {
-    "./**/*.(tsx|ts)": [
-      "tsdx lint --fix"
-    ]
-  },
-  "husky": {
-    "hooks": {}
-  },
-  "prettier": {
-    "printWidth": 80,
-    "semi": true,
-    "singleQuote": true,
-    "trailingComma": "es5",
-    "tabWidth": 2
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^4.11.4",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/styles": "^4.11.4",
+    "clsx": "^1.1.1",
+    "core-js": "^3.6.5",
+    "document-register-element": "1.13.1",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-is": "17.0.2",
+    "regenerator-runtime": "0.13.7",
+    "styled-components": "5.2.1",
+    "tslib": "^2.0.0"
   },
-  "name": "@prenda/spark",
-  "author": "Jake McBride",
-  "module": "dist/prenda-spark.esm.js",
-  "size-limit": [
-    {
-      "path": "dist/prenda-spark.cjs.production.min.js",
-      "limit": "10 KB"
-    },
-    {
-      "path": "dist/prenda-spark.esm.js",
-      "limit": "10 KB"
-    }
-  ],
   "devDependencies": {
-    "@babel/core": "^7.12.16",
-    "@size-limit/preset-small-lib": "^4.9.2",
+    "@babel/core": "7.12.13",
+    "@babel/preset-env": "7.12.13",
+    "@babel/preset-react": "7.12.13",
+    "@babel/preset-typescript": "7.12.13",
+    "@nrwl/cli": "12.2.0",
+    "@nrwl/eslint-plugin-nx": "12.2.0",
+    "@nrwl/jest": "12.2.0",
+    "@nrwl/linter": "12.2.0",
+    "@nrwl/react": "^12.2.0",
+    "@nrwl/storybook": "^12.2.0",
+    "@nrwl/tao": "12.2.0",
+    "@nrwl/web": "12.2.0",
+    "@nrwl/workspace": "12.2.0",
     "@storybook/addon-essentials": "^6.2.9",
-    "@storybook/addon-info": "^5.3.21",
+    "@storybook/addon-knobs": "^6.1.11",
```

Comment:
```suggestion
```

The knobs addon was superceded by args and controls which is under the essentials addon. See more at [nx docs](https://nx.dev/latest/angular/storybook/overview#storybook-v6-args-and-controls) and [storybook docs](https://storybook.js.org/docs/angular/essentials/controls).